Content.Server/BluespaceHarvester/BluespaceHarvesterRiftSystem.cs

```diff
@@ -0,0 +1,75 @@
+using Robust.Shared.Prototypes;
+using Robust.Shared.Random;
+using System.Linq;
+
+namespace Content.Server.BluespaceHarvester;
+
+public sealed class BluespaceHarvesterRiftSystem : EntitySystem
+{
+    [Dependency] private readonly IRobustRandom _random = default!;
+
+    private float _updateTimer = 0.0f;
+    private const float UpdateTime = 1.0f;
+
+    public override void Initialize()
+    {
+        base.Initialize();
+    }
+
+    public override void Update(float frameTime)
+    {
+        base.Update(frameTime);
+
+        _updateTimer += frameTime;
+        if (_updateTimer < UpdateTime)
+            return;
+
+        _updateTimer -= UpdateTime;
```

Comment:
Having a timer in the system means that all rifts will release monsters at the same moment. I would make each portal component have an AccumulatedTime field, and a SpawnInterval field. And the system loops through all the portals, checking to see if they have accumulated enough time to spawn. This will allow you to configure different portals different spawn interval, and make that the monsters will appear not one huge crowd, but one by one.